rtb/norme_di_progetto_ver0.6.2.typ

```diff
@@ -531,9 +556,192 @@ Il verificatore può consultare sia il documento in formato `.pdf` per maggiore
++ Il verificatore controlla il lavoro svolto:
+  - Se la verifica ha esito positivo:
+    + Il verificatore approva su GitHub la _pull request_ e chiede al responsabile di effettuare il _merge_ nel ramo principale .
+    + La _issue_ viene marcata "_done_" su GitHub automaticamente.
+  - Se la verifica ha esito negativo:
+    + Il verificatore rilascia una lista di cambiamenti suggeriti nella _pull request_.
+    + L'incaricato apporta le modifiche suggerite e si torna al punto 4.
+
+=== Coordinamento
+Il coordinamento è l'attività che riguarda la gestione delle comunicazioni e degli incontri tra i membri del _team_, proponenti e committenti del progetto. L'efficienza del progetto e il coinvolgimento di tutte le parti interessate dipendono dal coordinamento.
+
+
+La gestione della comunicazione interna ed esterna, la conduzione delle riunioni e la definizione di comportamenti comuni per i membri del _team_ sono esempi di attività di coordinamento.
+
+==== Comunicazioni
+===== Comunicazioni interne
+Le comunicazioni internte vengono gestite con due strumenti separati: Telegram per le comunicazioni rapide da dispositivi _mobile_ e Discord per le riunioni e le chiamate tra i membri del _team_.
+
+
+===== Comunicazioni esterne
+Il responsabile del progetto sarà incaricato di gestire il dialogo esterno attraverso l'indirizzo _email_ #formatLink(url: "mailto:sweat.unipd@gmail.com").
+Per quanto riguarda il mezzo di comunicazione per i SAL con i proponenti lo strumento utilizzato è _Google Meet_ che rende le videochiamate accessibili a tutti.
+
+==== Riunioni
+Al fine di garantire l'efficienza delle riunioni il responsabile corrente avrà il compito di interloquire con i proponenti per riepilogare i punti principali del periodo trascorso e esprimere dubbi e incertezze emerse dal gruppo.
+
+===== Riunioni Interne
+Le riunioni interne sono programmate di comune accordo tra i membri del gruppo. In caso di necessità è possibile richiedere riunioni straordinarie durante la settimana tramite il canale dedicato su Telegram con data e orario stabiliti attraverso un sondaggio. Tutte le riunioni _online_ si svolgeranno nel canale Discord appositamente designato. Nelle riunini interne l'obiettivo finale è quello di risolvere le mancanze segnalate dall'ordine del giorno, per cui nella fase finale della riunione il responsabile, dopo aver discusso con il gruppo, pianifica le attività da svolgere e crea rispettivamente le _issue_ come da procedura sopra.
+
+===== Riunioni Esterne
+Le riunioni esterne coinvolgono i membri del gruppo e il proponente.
+
+
+Per le riunioni con il proponente viene utilizzata la piattaforma Google Meet e il _link_ per accedere alla chiamata viene comunicato al _team_ di volta in volta. In accordo con l'azienda proponente gli incontri di fine _sprint_ avvengono ogni due settimane, in quanto lo _sprint_ dura due settimane, mentre a metà di ogni _sprint_ si svolgerà un incontro di allineamento più breve per un confronto e degli aggiustamenti.
```

Comment:
Allora... forse un po' troppo ripetitivo dire incontri di fine sprint ogni due settimane siccome durano due settimane. Da un lato ti direi anche ovvio. Se trovi un modo per modificarla, mi faresti un grandissimo favore. Provo a pensare anche io se mi viene in mente qualcosa